embrace-android-otel/src/main/kotlin/io/embrace/android/embracesdk/internal/otel/sdk/LimitsValidator.kt

```diff
@@ -0,0 +1,52 @@
+package io.embrace.android.embracesdk.internal.otel.sdk
+
+import io.embrace.android.embracesdk.internal.config.instrumented.OtelLimitsConfigImpl
+import io.embrace.android.embracesdk.internal.config.instrumented.schema.OtelLimitsConfig
+import io.embrace.android.embracesdk.spans.EmbraceSpanEvent
+
+class LimitsValidator(
```

Comment:
This class isn't only validating limits, but also other things like blank strings. Should we try to think of another name? Maybe something like TelemetryValidator, or SpanDataValidator?